tests/contrib/structlog/test_structlog_logging.py

```diff
@@ -0,0 +1,295 @@
+import json
+
+import pytest
+import structlog
+
+from ddtrace import config
+from ddtrace.constants import ENV_KEY
+from ddtrace.constants import SERVICE_KEY
+from ddtrace.constants import VERSION_KEY
+from ddtrace.contrib.structlog import patch
+from ddtrace.contrib.structlog import unpatch
+from tests.utils import DummyTracer
+
+
+cf = structlog.testing.CapturingLoggerFactory()
+
+
+def _test_logging(output, span, env="", service="", version=""):
```

Comment:
nit: I don't think we need these optional arguments. We can always pass in these values
```suggestion
def _test_logging(output, span, env, service, version):
```